packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -8060,7 +8060,7 @@ export class List<T = any> extends React.Component<IListProps<T>, IListState<T>>
     }
 
 // @public (undocumented)
-export const ListPeoplePicker: React.FunctionComponent<IPeoplePickerProps>;
+export const ListPeoplePicker: React.StatelessComponent<IPeoplePickerProps>;

```

Comment:
Do you know why this changed?

Reply:
It almost seems `FunctionComponent` was a bug of sorts. This type is more accurate with `styled`'s return type.

Reply:
(They are otherwise equivalent types.)